test/packages/npm-test-shrinkwrap/test.js

```diff
@@ -1,4 +1,4 @@
-var path = require("path")
```

Comment:
This file is a part of the legacy tests and... as best as I can tell shouldn't actually be in the branch at all? Regardless, the one that's used now is [test/tap/legacy-shrinkwrap.js](https://github.com/npm/npm/blob/lts/test/tap/legacy-shrinkwrap.js)